src/App.tsx

```diff
@@ -1,18 +1,22 @@
-import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Navigate, Outlet, useLocation } from 'react-router-dom';
 
 export const App = () => {
+  const { pathname } = useLocation();
+
+  if (pathname === '/home') {
+    return <Navigate to="/" replace />;
```

Comment:
The redirection from '/home' to '/' might not be the intended behavior. Ensure that this logic aligns with the application's routing requirements. If '/home' should not redirect to '/', consider adjusting the condition or the target path.

Reply:
This logic aligns with the application's routing requirements